JavaScript/Topics/9.Destructuring_Rest_Parameters_And_Spread_Syntax/README.md

```diff
@@ -0,0 +1,148 @@
+# Array Destructuring
+Destructuring assignment is a syntax that allows you to assign object properties or array items as variables. This can greatly reduce the lines of code necessary to manipulate data in these structures.
```

Comment:
The top heading should be `# JavaScript Destructing`

Reply:
I think destructuring is corry